doc/installing-spacegraphcats.md

```diff
@@ -117,6 +123,12 @@ This will take a few minutes.
 In the `spacegraphcats/` top level directory (containing e.g. `README.md`),
 run:
 
+```
+pip install -e .
+```
+
+This installs the spacegraphcats project in editable mode.
```

Comment:
maybe: "in developer mode, so that the installed package points at this directory and changes in this directory are available in the virtual environment."

Reply:
Updated.

Reply:
thank you!